### PR TITLE
Discuss testing app behavior in browser with ngrok

### PR DESCRIPTION
**Pull request by malandrina**
- https://trello.com/c/HTdqj5GB

### CODE REVIEW

File: book/testing/coffeescript_unit_tests.md [hunk 1]
```diff
@@ -66,6 +66,14 @@ geocoding is unsuccessful:
 
 ` spec/javascripts/spec_helper.js.coffee@e9f6739:25,33
 
+### Testing Behavior in the Browser
```
Comment (malandrina) [inline book/testing/coffeescript_unit_tests.md]: @joshuaclayton, any ideas for a better header?

Comment (joshuaclayton) [inline book/testing/coffeescript_unit_tests.md]: @malandrina what about "Verifying Reverse Geocoding in the Browser"? Mentioning "Testing" (not the automated kind) in a testing section seems a bit strange to me as well

**Comment by malandrina** [comment]
@joshuaclayton I went with "Validating Reverse Geocoding in the Browser" in https://github.com/thoughtbot/geocoding_on_rails/commit/1e385b86eb478dce39cacecead04562c0253679c. What do you think?

**Comment by joshuaclayton** [comment]
@malandrina :+1: that sounds great too

**Pull request merged**